README.md

```diff
@@ -1,52 +1,192 @@
-## BEGIN - TO DELETE TILL END
+# reshuffle-shopify-connector
 
-THIS IS A TEMPLATE REPO FOR NEW RESHUFFLE CONNECTORS
-1. Create a new connector repo from this template using this link https://github.com/reshufflehq/reshuffle-template-connector/generate
-2. Clone the repo locally
-3. Rename all occurrences of _CONNECTOR_NAME_
-4. `npm install`
-5. `npm run build:watch`
-6. Implement your events/actions in `src/index.ts`
-7. `npm run lint`
-8. Push your code
-9. Go to https://app.circleci.com/projects/project-dashboard/github/reshufflehq/
-    a. You should see your new connector repo
-    b. click on `Set Up Project` for the repo
-    c. click on `Use Existing Config`
-    d. click on `Start Building`
+[Code](https://github.com/reshufflehq/reshuffle-shopify-connector) |  [npm](https://www.npmjs.com/package/reshuffle-shopify-connector) | [Code sample](https://github.com/reshufflehq/reshuffle/tree/master/examples/shopify)
 
-10. If circle CI checks are all green, you are all set!
+`npm install reshuffle-shopify-connector`
 
-// Keep documentation template below
+This connector uses [Official Node Shopify Client](https://www.npmjs.com/package/shopify-api-node) package.
 
-## END
+### Reshuffle Shopify Connector
 
-# reshuffle-_CONNECTOR_NAME_-connector
+This connector provides a connector for [Shopify](https://www.shopify.com/online).
 
-### Reshuffle _CONNECTOR_NAME_ Connector
+The following example listen to Shopify events:
+```js
+const { Reshuffle } = require('reshuffle')
+const { ShopifyConnector } = require('reshuffle-shopify-connector')
 
-This connector provides <description>.
+const app = new Reshuffle()
+const connector = new ShopifyConnector(app, {
+  shopName: process.env.SHOPIFY_SHOP_NAME,
+  apiKey: process.env.SHOPIFY_API_KEY,
+  password: process.env.SHOPIFY_PASSWORD
+})
+  
+connector.on({ topic: 'orders/fulfilled' }, (event, app) => {
+  console.log('congratulations! Another order fulfilled')
```

Comment:
Suggestion: Maybe` console.log` some detail of the order (order number?)

Reply:
You caught me...
I didn't bother using the order api to create one and see what comes out to the event listener so I didn't know the structure of the data we get for new orders. But because it's just an example at the top of connector documentation, I kept it very simple